esp/esp/themes/theme_data/fruitsalad/templates/main.html

```diff
@@ -143,9 +143,6 @@
 <div class="corners"><div class="ul"></div><div class="ur"></div></div>
 {% if request.path|equal:"/teach/index.html" or request.path|equal:"/learn/index.html" %}
 
-{# announcements #}
-<div id="divannouncements">
-<ul>
 {% load preview %}
 
 {% if request.path|equal:"/teach/index.html" %}
```

Comment:
I think this should also be inside the conditional, since I think it's supposed to act similarly to announcements, although it's not used anymore (at least not by MIT).